classes/core/ValidationServiceProvider.php

```diff
@@ -176,12 +185,14 @@ protected function resolve(array $data, array $rules, array $messages, array $at
                              */
                             protected function getMessage($attribute, $rule)
                             {
+                                $this->rehydrateValidationErrorMessages();
+
```

Comment:
This is need to rehydrate the validation message with our custom rules related validation messages .